src/frontend/src/app/myprofile/myprofile.component.html

```diff
@@ -2,8 +2,9 @@
 <div align="center">
   <h1 class="title">CodeChamp</h1>
 </div>
-<div align="center">
-  <h1 class="title">Player1</h1>
+<div class="playerDiv" align="center">
```

Comment:
This might look better with a small margin around it

Reply:
Slightly increased margin-top